### PR TITLE
Fix Build Error

### PR DESCRIPTION
**Pull request by deepakwadhwani**
Fixing Build Error introduced as part of https://github.com/caskdata/cdap/pull/8666.

### CODE REVIEW

File: cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala [hunk 1]
```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed", programRunId.getProgram)
```
Comment (chtyim) [inline cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: So we don't like the sentence?

Comment (deepakwadhwani) [reply cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: Removing the '.' after the error and the Run ID which is included in the "started" message. Also fixing the build error.

Comment (deepakwadhwani) [reply cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: @chtyim I have reverted any change to the log. Just fixing the build error here.

File: cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala [hunk 2]
```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun:Any)
```
Comment (chtyim) [inline cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: What is `:Any`??

Comment (deepakwadhwani) [reply cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: "Any" forces Scala to choose one between the two methods. Here is the error: 
build	27-Apr-2017 19:12:07	[ERROR] /var/bamboo/xml-data/build-dir/CDAP-DRC-JOB1/cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala:217: error: ambiguous reference to overloaded definition,
build	27-Apr-2017 19:12:07	[ERROR] both method info in trait Logger of type (x$1: String, x$2: <repeated...>[Object])Unit
build	27-Apr-2017 19:12:07	[ERROR] and  method info in trait Logger of type (x$1: String, x$2: Any, x$3: Any)Unit
build	27-Apr-2017 19:12:07	[ERROR] match argument types (String,String,String)
build	27-Apr-2017 19:12:07	[ERROR]           LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
build	27-Apr-2017 19:12:07	[ERROR]               ^
build	27-Apr-2017 19:12:08	[ERROR] one error found

Comment (chtyim) [reply cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: Oh, it's Scala. You can do `Array(programRunId.getProgram, programRunId.getRun)`, which I think is cleaner.

Comment (deepakwadhwani) [reply cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala]: @chtyim Using Array not give the desired log. 
"Spark program [PageRankSpark, 0c152701-2b8f-11e7-87b8-42010afa0004] completed. Program RunID: [{}]"
Can you please take another look.

**Comment by chtyim** [comment]
LGTM

**Pull request merged**